inlong-sort/sort-connectors/pom.xml

```diff
@@ -40,7 +40,9 @@
         <module>kafka</module>
         <module>jdbc</module>
         <module>pulsar</module>
+
         <module>iceberg</module>
+        <module>iceberg-dlc</module>
```

Comment:
Maybe it is better to call dlc?

Reply:
tdsql is call TDSQLPostgres. It's Similar